content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -86,7 +86,7 @@ The different :guilabel:`Program Types` available on the program form are:
 - :guilabel:`Next Order Coupons`: Generate and share single-use coupon codes that grant access to
   rewards on the customer's next order.
 - :guilabel:`Loyalty Cards`: When making purchases, the customer accumulates points to exchange for
-  rewards on future orders.
+  rewards on current and/or future orders.
```

Comment:
I thought we were supposed to use just "and" instead of "and/or" according to the style guide?

Reply:
Oh, really? I am not aware of that. Also, in this case, the statement requires the use of it. So, with that in mind, I am going to proceed with that wording -- but I'll be sure to keep it in mind moving forward. Thanks!